src/prefect/tasks.py

```diff
@@ -51,7 +51,8 @@
     from prefect.context import TaskRunContext
 
 
-T = TypeVar("T")  # Generic type var for capturing the inner return type of async funcs
+# Generic type var for capturing the inner return type of async funcs
+T = TypeVar("T")
```

Comment:
[As Zanie mentioned](https://github.com/PrefectHQ/prefect/pull/9692#discussion_r1213192612), this might have been accidentally modified?

Reply:
Yup, reverting this too